addons/point_of_sale/static/src/js/screens.js

```diff
@@ -1632,6 +1633,30 @@ var ReceiptScreenWidget = ScreenWidget.extend({
             this.lock_screen(false);
         }
     },
+    send_receipt_to_customer: function(order, orderServerId) {
```

Comment:
mover send_receipt_to_customer -> PaymentScreen
remove order -> this.pos.get_order
rename orderServerId -> invoice_server_id